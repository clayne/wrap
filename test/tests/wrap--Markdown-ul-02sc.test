wrap | /dev/null | -u | md-ul-02sc.md | 0
