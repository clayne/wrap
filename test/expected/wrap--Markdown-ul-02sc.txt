+ List item #1.

+ List item #2.

Not part of the list.

+ Another list item #1.

+ Another list item #2.
